### DISCUSSION TITLE
How to hide combi mode in rofi

### DISCUSSION THREAD

**Discussion by W0NB0N** [davatorium/rofi#1986] [Q&A] [answered]
I have removed combi from the `modi: "run,drun,filebrowser"` but the combi mode still shows up while opening up rofi

**Comment by DaveDavenport** [comment]
It should not, combi is disabled by default unless you pass it by 'modi', or you launch it via `-show combi` .

**Reply by W0NB0N** [reply, marked as answer]
my i3 config was launching it with `-show combi`, changing that worked. Thanks a lot!
